include/oneapi/dpl/pstl/glue_memory_impl.h

```diff
@@ -187,13 +187,13 @@ uninitialized_fill_n(_ExecutionPolicy&& __exec, _ForwardIterator __first, _Size
 
 #if (_PSTL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN || _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN)
 
-const oneapi::dpl::execution::parallel_policy&
+inline const oneapi::dpl::execution::parallel_policy&
```

Comment:
I hope we don't have a similar situation with `__select_backend`, do we?

Reply:
The `__select_backend` functions is template functions so there are no problems for them.

Reply:
That's what I was asking for. Thanks!